lit_tests/kernel/wave/attention/extend_attention.py

```diff
@@ -70,7 +70,7 @@ def test_extend_attention():
     # CHECK-LABEL: test_extend_attention
     # CHECK-DAG:            #[[map0:.*]] = affine_map<()[s0] -> (s0 ceildiv 64)>
     # CHECK-DAG:            #[[map1:.*]] = affine_map<()[s0] -> (s0 * 16)>
-    # CHECK:              stream.executable.export public @extend_attention workgroups(%[[ARG0:.+]]: index, %[[ARG1:.+]]: index, %[[ARG2:.+]]: index, %[[ARG3:.+]]: index)
+    # CHECK:              stream.executable.export public @extend_attention workgroups(%[[ARG0:.+]]: index, %[[ARG1:.+]]: index, %[[ARG2:.+]]: index, %[[ARG3:.+]]: index, %[[ARG4:.+]]: index)
```

Comment:
Could you add another lit test with the custom mask so we know what the IR changes are?

Reply:
Done